app/services/doc_auth/result_codes.rb

```diff
@@ -0,0 +1,36 @@
+module DocAuth
+  module ResultCodes
```

Comment:
if we got a new doc auth vendor tomorrow, they would probably not use these result codes right? like these happened to be the same across Acuant + TrueID, since they're still used in TrueID, would it be better move these to the LexisNexis namespace?

Ditto for `IssuerTypes`

```suggestion
  module LexisNexis
    module ResultCodes
```